From 65f75d1cbbb159c824ea730dec3407f92b836cc9 Mon Sep 17 00:00:00 2001
From: "SANCRIS\\joeltovar" <joeltovar@imb-pc.edu.gt>
Date: Fri, 11 Apr 2025 16:56:56 -0600
Subject: [PATCH] Firs commit

---
 App.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/App.js b/App.js
index 09f879b..c562ad4 100644
--- a/App.js
+++ b/App.js
@@ -4,7 +4,7 @@ import { StyleSheet, Text, View } from 'react-native';
 export default function App() {
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+      <Text>Hello World!</Text>
       <StatusBar style="auto" />
     </View>
   );
-- 
2.40.0.windows.1

